### ISSUE TITLE
Doesn't rename pages when using Lister Pro

### ISSUE THREAD

**Issue by andoro** [adrianbj/PageRenameOptions#2]
Hello Adrian,

I just realized, that when I'm batch editing pages using the inline editor and listed with Lister Pro, the page name (URL segment) doesn't rename.
When I'm opening in a popup  the page editor one by one it works. Is this a bug or it's by design?

**Comment by adrianbj** [comment]
Not really a bug, but rather a limitation due to how it renames the page. That said, please try the version from the second last commit - it still had the option to handle pages re-titled via the API. If that works for ListerPro then it should be possible to add that back in for this situation.

**Comment by andoro** [comment]
Thanks for quick answer!
Sorry, what do you mean? " try the version from the second last commit"

**Comment by adrianbj** [comment]
This version here: https://github.com/adrianbj/PageRenameOptions/tree/52e387b8ffe10a7a06f7b4787b7f36d9005cf946

**Comment by andoro** [comment]
I tried it, enabled API rename option but it doesn't work.
Just to make sure we discuss about the same thing.
Here:
![image](https://user-images.githubusercontent.com/775765/89338561-09274700-d69d-11ea-9c21-c87a7ffa603e.png)
I'm expecting to see the URL is updating based on the page name when clicking Save icon on the right of that line.

**Comment by adrianbj** [comment]
Yeah, I do understand what you want - I've just never tested with ListerPro. The renaming in this module (the current version) is all done with JS in the page editing interface so it just doesn't support ListerPro as it is. It might be possible to support it, but I don't have time at the moment unfortunately. I'll revisit when I have some time, or if you can implement it, please submit a PR.
